.github/workflows/regen-abidump.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+
+./.github/workflows/posix-deps-apt.sh
```

Comment:
This asks me where I live when it installs tzdata. That shouldn't be necessary.
```suggestion
export DEBIAN_FRONTEND=noninteractive
./.github/workflows/posix-deps-apt.sh
```